src/index.html

```diff
@@ -4,9 +4,374 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="/" class="nav__item is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/"
+            class="nav__item"
+            data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Фото
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="/" class="nav__item">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="catalog__cards">
+          <div class="card catalog__card" data-qa="card">
+            <img
+              src="./images/imac.jpeg"
+              alt="imac"
+              class="card__img"
+            >
+            <p class="card__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </p>
+            <p class="card__code">
+              Код товара: 195434
+            </p>
+            <div class="card__starsWrapper">
```

Comment:
don't use camel case on bem
```suggestion
            <div class="card__stars-wrapper">
```